docs/templates/component/docs/ColorTheming.md

```diff
@@ -0,0 +1,47 @@
+### Theming
+
+You can theme a <#component_name#> with your app's color scheme using the ColorThemer extension.
+
+You must first add the Color Themer extension to your project:
+
+``` bash
+pod 'MaterialComponents/<#component#>+Extensions/ColorThemer'
+```
+
+You can then import the theming APIs:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+import MaterialComponents.Material<#component#>_ColorThemer
```

Comment:
I feel like this red is very intrusive, could we have it the same color scheme as the other wildcards?